modules/cnf-image-based-upgrade-generate-seed-image.adoc

```diff
@@ -48,7 +48,19 @@ generators:
 - example-target-sno3.yaml
 ----
 
-... Commit the `kustomization.yaml` changes in your Git repository and push the changes.
+... If you have a `kustomization.yaml` that references one `SiteConfig` CR, remove your seed cluster's `SiteConfig` CR from the `kustomization.yaml` and add the `commonLabels: {}` line:
```

Comment:
In the bug there are two possibilities for updating the CR when only one SiteConfig is given. The "add generators: {}" version is better aligned with the changes that the customer is making (vs adding a new field "commonLabels").

Reply:
Understood! I removed `commonLabels: {}`, updated the `generators: {}` line both in the instructions and in the codeblock.